src/MagnumPlugins/AssimpImporter/AssimpImporter.h

```diff
@@ -307,6 +341,9 @@ enum. Some of them are enabled by default, some not; options for not yet
 supported features are omitted. These are passed to Assimp when opening a file,
 meaning a change in these will be always applied to the next opened file.
 
+Additional options exist to configure animation import. See
+below for all options and their default values.
```

Comment:
Since the options are already referenced from the "Animation import" section above, I don't think this paragraph is needed.

Reply:
Done